assets/css/bootstrap-shims.css

```diff
@@ -539,6 +539,18 @@ table.table-striped tbody tr:nth-child(even) {
   @apply bg-gray-100 dark:bg-gray-700;
 }
 
+table.table-striped-schedule tbody tr:nth-child(even) {
```

Comment:
This bootstrap-shims CSS file is effectively deprecated and intended to only shim existing required bootstrap styles leftover from the bootstrap -> tailwind transition. At some point, it could be removed as technical debt cleanup.

Could we apply these tailwind styles directly to the elements that use them? Or, if not, can we put them in a more appropriate CSS file?

Reply:
Moved the styles to an appropriate CSS file